### PR TITLE
fix code tab spacing

### PR DESCRIPTION
**Pull request by Kagre**

### CODE REVIEW

**Comment by PRMerger9** [comment]
@Kagre : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/odbc/reference/appendixes/date-time-and-timestamp-escape-sequences.md [hunk 1]
```diff
@@ -28,22 +28,51 @@ ODBC defines escape sequences for date, time, and timestamp literals. The syntax
...
+
 ODBC-esc-terminator ::= }  
+
 date-value ::=   
-     years-value date-separator months-value date-separator days-valuetime-value ::=   
-     hours-value time-separator minutes-value time-separatorseconds-valuetimestamp-value ::= date-value timestamp-separator time-valuedate-separator ::= -  
+     years-value date-separator months-value date-separator days-value
+
+time-value ::=   
+     hours-value time-separator minutes-value time-separatorseconds-value
```
Comment (David-Engel) [inline docs/odbc/reference/appendixes/date-time-and-timestamp-escape-sequences.md:54]:
```suggestion
     hours-value time-separator minutes-value time-separator seconds-value

```

Comment (ktoliver) [reply docs/odbc/reference/appendixes/date-time-and-timestamp-escape-sequences.md]: Done

**Comment by ktoliver** [comment]
@David-Engel I went ahead and added the space where indicated. The PR is otherwise approved, it looks like, so I'll merge it now. Thanks, all.

**Pull request merged**